### ISSUE TITLE
Should cover lifestyle creep

### ISSUE THREAD

**Issue by snoop0x7b** [emilepetrone/financial_advice#4]
Lifestyle creep is a phenomenon where people gain income but don't end up saving more because they spend more on luxuries. https://www.investopedia.com/terms/l/lifestyle-creep.asp#:~:text=Lifestyle%20creep%20occurs%20when%20an,former%20luxuries%20become%20new%20necessities.&text=A%20hallmark%20of%20lifestyle%20creep,right%20rather%20than%20a%20choice.

**Comment by emilepetrone** [comment]
Great idea +1

**Comment by lenabuilds** [comment]
Yeah, especially how you do budgeting and planning. And would love to hear your opinions on real estate investment if you have any!
